INSTALL.md

```diff
@@ -197,7 +197,21 @@ These need to be manually installed before rofi can be installed either via sour
 
 ### Fedora
 
-rofi from [russianfedora repository](http://ru.fedoracommunity.org/repository)
+You can build it on Fedora 27 or 28, you would need to have these external libraries beside the build packages that were mentioned above: 
```

Comment:
it's not "*beside*", it's the packages that are mentioned above, the names are just different in fedora repositories. I would rephrase as

> You can follow these instructions to build rofi on Fedora, the packages name are different in the Fedora repositories:

* pango-devel
* cairo-devel
* ...